SECURITY.md

```diff
@@ -2,6 +2,18 @@
 
 ## Secret Detection
 
+Secret Detection aims to prevent the unintentional leak of sensitive information including: passwords, authentication tokens, and private keys. It checks source files and configuration files to detect well-known and common patterns that look like secrets or credentials and reports findings that are potentially risky to share.
+
+### Goal
+
+Overall we want to help developers write better code and worry less about common security mistakes. Our goal is to provide Secret Detection as a part of the standard software development lifecycle (SDLC). This means that Secret Detection is executed every time a new commit is pushed to a branch.
+
+This package is designed with the enterprise client in mind: providing a backwards compatible, systematic means of:
+
+- Preventing new secrets from entering the code base,
+- Detecting if such preventions are explicitly bypassed, and
+- Providing a checklist of secrets to roll, and migrate off to a more secure storage.
```

Comment:
i really like this addition of rolling over detected secrets. are you alluding to the baseline file here? if so i think the checklist should be made more clear

Reply:
Appended not to end of line.  The secrets baseline is the primary feature that spans across the tooling.